src/actions/index.js

```diff
@@ -22,6 +22,18 @@ export const connectToServer = ({ username, hostname, port }) => (dispatch, getS
         contents,
       })
       handleWebsocketsConnection(dispatch, getState)
+
+      AsyncStorage.getItem(`${username}-favorites`).then(favoritesString => {
```

Comment:
I see that there is a `const getFavoritesKey = username => `${username}-favorites`` in the favoritesMiddleware, could be best to centralize that.